tcp_check/tests/test_tcp_check.py

```diff
@@ -5,14 +5,18 @@
 import platform
 import re
 import socket
+from collections import namedtuple
 from copy import deepcopy
 
 import mock
+import pytest
 
 from datadog_checks.tcp_check import TCPCheck
 
 from . import common
 
+addr_tuple = namedtuple('addr_tuple', ['address', 'socket_type'])
```

Comment:
This will go away if we apply the previous suggestion of making this a module-level type / class.